src/input/test/browser/camera_access.spec.ts

```diff
@@ -40,6 +40,34 @@ describe('CameraAccess (browser)', () => {
             expect(v[0].kind).to.equal('videoinput');
             expect(v[0].label).to.equal('fake_device_0');
         });
+
+        it('works with constraints (no filtering)', async () => {
+            // With constraints that match the fake device, should return the same device
+            const v = await Quagga.CameraAccess.enumerateVideoDevices({ width: 320, height: 240 });
```

Comment:
ok, i need an explanation here -- is cypress providing a fake camera device?  i'm guessing it must have, since apparently i wrote a test a long time ago that checked it's existence, directly above this one, but I absolutely do not remember it.  
You may want to explain that a little better with the comment, though, expand upon "the fake device".